test/data/testSP.py

```diff
@@ -151,6 +154,10 @@ def testscftargetdim(self):
         """Do the scf targets have the right dimensions?"""
         self.assertEquals(self.data.scftargets.shape, (len(self.data.scfvalues), len(self.data.scfvalues[0][0])))
 
+    def testscftargets(self):
+        """Are correct number of SCF convergence criteria being parsed?"""
+        self.assertEquals(len(self.data.scftargets[0]), self.num_scf_criteria)

```

Comment:
You could check `scftargets[i]` for all possible `i`, no?

Reply:
In SP calculations we get `len(scftargets) = 1` since we only have one iteration of SCF energy calculation. So `i = 1` and we only need to check once.

Reply:
Duh, this is for SP. Perhaps check the length for good form.

Reply:
Yeah. That would make sense.